.github/workflows/ci.yml

```diff
@@ -156,7 +156,7 @@ jobs:
         run: src/ci/scripts/verify-stable-version-number.sh
         if: success() && !env.SKIP_JOB
       - name: run the build
-        run: src/ci/scripts/run-build-from-ci.sh
+        run: src/ci/scripts/run-build-from-ci.sh 2>&1
```

Comment:
Instead of repeating this everywhere the script is called, it should also work to just put `exec 2>&` at the top of the script, I think.

Reply:
This file is auto-generated, so repeating is not that problematic, I think :) In the source file (`ci.yml`) it is just once. I find it a bit cleaner to do this from the outside rather than to hardcode it in the script. Someone might want to execute it locally.

Reply:
> This file is auto-generated,

Ah, I was already wondering why the comment only got added 1 out of 4 times.^^